barretenberg/ts/scripts/build_wasm.sh

```diff
@@ -4,8 +4,8 @@ set -e
 if [ -z "$SKIP_CPP_BUILD" ]; then
   # Build the wasms and strip debug symbols.
   cd ../cpp
-  cmake --preset wasm-threads && cmake --build --preset wasm-threads
```

Comment:
I made it a bit easier to work on command line by suppressing the very verbose CMake configure output. You'll still get a failure message if something goes wrong.